tests/NHSD.BuyingCatalogue.Solutions.Persistence.DatabaseTests/SolutionDetailRepositoryTests.cs

```diff
@@ -563,15 +521,11 @@ await CatalogueItemEntityBuilder
 
             await SolutionEntityBuilder.Create()
                 .WithId(Solution1Id)
-                .Build()
-                .InsertAsync();
-
-            await SolutionDetailEntityBuilder.Create()
                 .WithImplementationTimescales(expectedImplementationTimescalesString)
                 .Build()
                 .InsertAndSetCurrentForSolutionAsync();
```

Comment:
As above, we should call `InsertAsync` instead of `InsertAndSetCurrentForSolutionAsync`.

Reply:
Replaced `InsertAndSetCurrentForSolutionAsync` with `InsertAsync`